src/libs/API.js

```diff
@@ -4,6 +4,20 @@ import * as Request from './Request';
 import * as SequentialQueue from './Network/SequentialQueue';
 import {version} from '../../package.json';
 
+/**
+ * Function for WRITE commands.
+ * All calls to API.write() will be persisted to disk as a JSON object.
+ * That object will have all the data needed to make the WRITE request later. In other words, we'll queue the WRITE
+ * request so that if the network is unavailable or the app is closed, we can send the WRITE request whenever the
+ * network comes back.
+ *
+ * @param {String} command - Name of API command to call
+ * @param {Object} apiCommandParameters - The object of parameters to send to the API
+ * @param {Object} onyxData  - Object containing errors, loading states, and optimistic UI data
```

Comment:
Would this be more "Object containing data that will be merged into onyx depending on the result of the request" or something similar?

Reply:
I was trying to generalize a bit more about what this object contains. Also, not all the data will be merged depending on the result, for example `onyxData.optimisticData` will be merged right away without knowing if the request is successful or not.

Maybe we can change it to something like: `Object containing errors, loading states, and optimistic UI data that will be merged into Onyx before and after a request is made`, thoughts?

Reply:
I updated this part with a more specific description